gbfs.md

```diff
@@ -625,11 +629,17 @@ Field Name | REQUIRED | Type | Defines
         "parking_type": "underground_parking",
         "parking_hoop": false,
         "contact_phone": "+33109874321",
-        "is_charging_station": true,
-        "vehicle_type_dock_capacity": {
-          "abc123": 7,
-          "def456": 9
-        }
+        "is_charging_station": "true",
```

Comment:
did you mean to stringify `true` here?

Reply:
Dumb question, but why not fix it now?  Isn't it better to be precise on what we are voting for?